src/framework/pages/utils/TimeManager.java

```diff
@@ -0,0 +1,66 @@
+package framework.utils;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+/**
+ * 
+ * @author Eliana Navia
+ *
+ */
+public class TimeManager {
+	static Calendar calendar = new GregorianCalendar();
+	static SimpleDateFormat simpleDateFormatHm = new SimpleDateFormat("HH:mm"); 
+	static Date date = new Date();
+	static String currentTime = "";
+
+
+	/**
+	 * 
+	 * @param baseDate 
+	 * @param simpleDateFormat
+	 * @param dateParameter
+	 * @param value
+	 */
+	public static String addRestDate(Date baseDate, SimpleDateFormat simpleDateFormat, String dateParameter, int value){
+		calendar.setTime(baseDate);
+		String newData = "";
+		switch(dateParameter){
+		case "month":
+			calendar.add(Calendar.DAY_OF_MONTH, value);
+			break;
+		case "hour": 
+			calendar.add(Calendar.HOUR_OF_DAY, value); 
+			break;
+		case "minute": 
+			calendar.add(Calendar.MINUTE, value); 
+			break;
+		default: 
+			calendar.add(Calendar.MINUTE, value); 
+			break;
+		}
+		newData =  simpleDateFormat.format(calendar.getTime()); 
+		return newData;
+	}	
+
+	public static String getLeastFifteenMinutes(){
```

Comment:
The "{" should be have a space before, please review that on all of your methods